src/core/withExperiment.js

```diff
@@ -1,12 +1,15 @@
 /* @flow */
+import config from 'config';
 import * as React from 'react';
 import invariant from 'invariant';
 import cookie from 'react-cookie';
 
+import log from 'core/logger';
 import { getDisplayName } from 'core/utils';
 
 export type WithExperimentInjectedProps = {|
-  variant: string,
+  experimentEnabled: boolean,
+  variant: string | null,
```

Comment:
I feel like this should be undefined (string | void) if not being used, no?

Reply:
if it is undefined, we cannot determine whether the prop has been passed or forgotten.

Reply:
mmm..ok..